Demo/HYPFormsTests/HYPFormsCollectionViewDataSourceTests.m

```diff
@@ -82,6 +82,50 @@ - (void)testIndexInForms
     [self.dataSource processTarget:[HYPFormTarget showFieldTargetWithID:@"last_name"]];
 }
 
+- (void)testEnableAndDisableTargets
+{
+    HYPFormField *targetField = [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
+
+    XCTAssertFalse(targetField.disabled);
+
+    HYPFormTarget *disableTarget = [HYPFormTarget disableFieldTargetWithID:@"display_name"];
+
+    [self.dataSource processTarget:disableTarget];
+
+    [self.manager fieldWithID:@"display_name" includingHiddenFields:YES];
```

Comment:
I'm not very familiar with HYPForms, but why do you need this line? Does it do some kind of processing? If it's only about gaining a pointer to a field (like on line 87), then it seems to be unnecessary, not to mention it's return value isn't used. I might easily be wrong though... :)